src/shared/ui/Preloader/Preloader.js

```diff
@@ -0,0 +1,16 @@
+import classes from './Preloader.module.scss';
+
+/**
+ * @function Preloader
+ * @returns {JSX.Element}
+ */
+
+export const Preloader = () => {
```

Comment:
Чтобы управлять состоянием прелоудера добавили ему пропс - isActive
```suggestion
export const Preloader = (props) => {
  if (!props.isActive) return null;
```